services/src/main/scala/cromwell/services/MetadataServiceActor.scala

```diff
@@ -28,7 +27,11 @@ object MetadataServiceActor {
   trait MetadataServiceAction extends MetadataServiceMessage with ServiceRegistryMessage {
     def serviceName = MetadataServiceName
   }
-  case class PutMetadataAction(event: MetadataEvent) extends MetadataServiceAction
+  object PutMetadataAction {
+    def apply(event: MetadataEvent) = new PutMetadataAction(List(event))
+    def apply(events: MetadataEvent*) = new PutMetadataAction(events)
```

Comment:
Yeah. While I don't have a citation for all of the benefits, I feel like I've seen the overload pattern as:

```scala
def apply(event: MetadataEvent) = new PutMetadataAction(List(event))
def apply(firstEvent: MetadataEvent, otherEvents: MetadataEvent*) = //…
```

Something like that would at minimum stop one from passing in "no events".